src/decouplededitoruiview.js

```diff
@@ -43,12 +44,16 @@ export default class DecoupledEditorUIView extends EditorUIView {
 		this.toolbar = new ToolbarView( locale );
 
 		/**
-		 * The editable of the decoupled editor UI.
+		 * The editables of the decoupled editor UI.
 		 *
 		 * @readonly
 		 * @member {module:ui/editableui/inline/inlineeditableuiview~InlineEditableUIView}
 		 */
-		this.editable = new InlineEditableUIView( locale, editableElement );
+		this.editables = editables.map( item => {
```

Comment:
I created a [ticket about it](https://github.com/ckeditor/ckeditor5/issues/1445) but at the moment `this.editable` is part of the interface (even if almost no-one uses it). So I think you should keep `editable` attached to the `main` root. However, you may have also additional `editables` properties with all editables, if it is useful.

Reply:
Yes, I was thinking about it too (in https://github.com/ckeditor/ckeditor5-editor-decoupled/issues/21#issuecomment-451102011):

> I was thinking if `editor.ui.view.editable.element` could be an "alias" for `editor.ui.view.editables[ 0 ].element` to keep backwards compatibility, but didn't implemented it.